tensorflow/core/grappler/optimizers/generic_layout_optimizer.h

```diff
@@ -46,6 +48,7 @@ class GenericLayoutOptimizer : public GraphOptimizer {
  private:
   RewriterConfig::Toggle opt_level_;
   RewriterConfig::CpuLayout cpu_layout_conversion_;
+  string enforced_layout_;
```

Comment:
Can this be "const string" instead?

Reply:
Just requires some refactoring of the ctors but sure, adding it!